homeassistant/components/tibber/sensor.py

```diff
@@ -88,7 +88,7 @@
         name="accumulated consumption",
         device_class=DEVICE_CLASS_ENERGY,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
-        state_class=STATE_CLASS_TOTAL_INCREASING,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Shouldn't we use state class total?

https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes

Reply:
👍